lenovo/yoga/6/13ALC6/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, pkgs, ...  }:
+
+{
+  imports = [
+    ../../../thinkpad/yoga.nix
+    ../../../../common/gpu/amd/default.nix
+  ];
+
+  boot.initrd.kernelModules = [ "ideapad_laptop" ];
+  hardware.opengl.extraPackages = with pkgs; [
+    vaapiVdpau
+    libvdpau-va-gl
+  ];
+
+  # Latest Kernel
+  boot.kernelPackages = lib.mkDefault pkgs.linuxPackages_latest;
+
+  # energy savings
+  hardware.bluetooth.powerOnBoot = lib.mkDefault false;
+  services.power-profiles-daemon.enable = false;
+  services.tlp.enable = lib.mkDefault true;
```

Comment:
Can you add a specific comment here why tlp was chosen and not the power-profiles-daemon?
Usually we also enable tlp using through the laptop module: `common/pc/laptop/default.nix` because this still allows gnome user to use power-management that is integrated with UI.

Reply:
i am a gnome user and i found tlp to be more powerful at saving power for this device than p-p-d
the goal of buying this device is to saveing power and get at least 10 hours, but p-p-d does not do this

Reply:
It is fair enough if this is your preference but we also have other users and having this enable unconditionally had led to evaluation errors. This why we introduced this module. If you have your own preferences, you can put those in your own configuration but we should try to have settings that work for everyone.